cmd/snap/cmd_create_cohort.go

```diff
@@ -0,0 +1,85 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"github.com/jessevdk/go-flags"
+	"gopkg.in/yaml.v2"
+
+	"github.com/snapcore/snapd/i18n"
+)
+
+var shortCreateCohortHelp = i18n.G("Create cohort keys for a series of snaps")
+var longCreateCohortHelp = i18n.G(`
+The create-cohort command creates a set of cohort keys for a given set of snaps.
+
+A cohort is a view or snapshot of a snap's "channel map" at a given point in
```

Comment:
Should we avoid the word "snapshot" here, not to create confusion wrt actual "snapshots"?

Reply:
I think it saying "view or snapshot" avoids the confusion.

If it doesn't, we also need to change [the docs](https://forum.snapcraft.io/t/managing-cohorts/8995).

Reply:
I see, although IMHO it's still an "OR", so a bit confusing. But it's a tiny detail which we can quickly fix afterwards if we decide so.